mkosi/resources/serial_getty_autologin.conf

```diff
@@ -1,5 +1,5 @@
 [Service]
 ExecStart=
-ExecStart=-/sbin/agetty -o '-p -- \\u' --autologin root --keep-baud 115200,57600,38400,9600 - $TERM
+ExecStart=-/sbin/agetty -o '-f -p -- \\u' --autologin root --noclear --keep-baud 115200,57600,38400,9600 - $TERM
```

Comment:
`--noclear` is new here, was that intentional? If so please mention in the commit message

Reply:
Dropped until I figure out why all the other getty units have it and serial-getty doesn't.